tests/integration/targets/ios_user/tests/cli/auth.yaml

```diff
@@ -72,6 +80,7 @@
       cisco.ios.ios_command:
         commands:
           - show version
+      ignore_errors: true
```

Comment:
this is not correct, as we now ignore the failure. We should uses SSH like above

Reply:
yes I am looking at an alternate way to check this is giving auth_failure